init.tf

```diff
@@ -8,7 +8,7 @@ locals {
 
   acmekv = var.department == "sds" ? "dtssds" : "dcdcftapps"
 
-  key_vault_environment = (var.environment == "sbox") ? "sandbox" : "${(var.environment == "perftest") ? "test" : "${var.environment}"}"
+  key_vault_environment = (var.environment == "sbox") ? "sandbox" : "${(var.environment == "perftest") ? "perftest" : "${var.environment}"}"
```

Comment:
Can we just remove reference to perftest if not needed?

Reply:
though, the KV looks to be named `acmedcdcftappstest `

Reply:
perftest reference removed. 
KV name is correct.  issue is that perftest cert use 'perftest' in the name.